btree/btree.c

```diff
@@ -2228,17 +2262,17 @@ static void get_next_rec_to_add(struct nd *node, uint64_t *key,  uint64_t *val)
 	ksize = h->ff_ksize;
 	proposed_key = rand();
 
-	find_key.k_data.ov_vec.v_nr    = 1;
-	find_key.k_data.ov_vec.v_count = &ksize;
-	find_key.k_data.ov_buf         = &p_key;
+	find_key.k_data.ov_vec.v_nr            = 1;
+	find_key.k_data.ov_vec.v_count         = &ksize;
+	find_key.k_data.ov_buf                 = &p_key;
 
-	slot.s_rec.r_key.k_data.ov_vec.v_nr = 1;
+	slot.s_rec.r_key.k_data.ov_vec.v_nr    = 1;
 	slot.s_rec.r_key.k_data.ov_vec.v_count = &ksize;
-	slot.s_rec.r_key.k_data.ov_buf = &p_key;
+	slot.s_rec.r_key.k_data.ov_buf         = &p_key;
 
-	slot.s_rec.r_val.ov_vec.v_nr = 1;
-	slot.s_rec.r_val.ov_vec.v_count = &vsize;
-	slot.s_rec.r_val.ov_buf = &p_val;
+	slot.s_rec.r_val.ov_vec.v_nr           = 1;
+	slot.s_rec.r_val.ov_vec.v_count        = &vsize;
+	slot.s_rec.r_val.ov_buf                = &p_val;
```

Comment:
`M0_BUFVEC_INIT_BUF` can be used for initializing `struct m0_bufvec` type variable.

Reply:
👍